edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -267,6 +267,7 @@ def _verify_jwt_signature(token, jwt_issuer, decode_symmetric_token):
         raise jwt.InvalidTokenError(exc_info[2]) from token_error
 
 
+# pylint: disable=R0917
```

Comment:
Is this one still needed after updating pylintrc?

Reply:
Yes, it suppresses `too-many-positional-arguments` error. 

![Screenshot 2024-10-23 at 8 30 38 AM](https://github.com/user-attachments/assets/37de3ab8-204c-4f25-a251-702239a8b706)

Reply:
Let's add `too-many-positional-arguments` in `pylintrc_tweaks` disable section instead of within the file.